src/pages/Overview/OverviewPage.tsx

```diff
@@ -130,41 +130,43 @@ class OverviewPage extends React.Component<OverviewProps, State> {
         health: r
       }));
     });
-    Promise.all(promises).then(responses => {
-      const allNamespaces: NamespaceInfo[] = [];
-      responses.forEach(response => {
-        const info: NamespaceInfo = {
-          name: response.namespace,
-          inError: [],
-          inWarning: [],
-          inSuccess: [],
-          notAvailable: []
-        };
-        const { showInError, showInWarning, showInSuccess, noFilter } = OverviewPage.summarizeHealthFilters();
-        let show = noFilter;
-        Object.keys(response.health).forEach(item => {
-          const health: Health = response.health[item];
-          const status = health.getGlobalStatus();
-          if (status === FAILURE) {
-            info.inError.push(item);
-            show = show || showInError;
-          } else if (status === DEGRADED) {
-            info.inWarning.push(item);
-            show = show || showInWarning;
-          } else if (status === HEALTHY) {
-            info.inSuccess.push(item);
-            show = show || showInSuccess;
-          } else {
-            info.notAvailable.push(item);
+    Promise.all(promises)
```

Comment:
Mmm is this blocked changed ?
I only see the a minor reformat but code seems similar.

Reply:
Prettier changed the indentation because I added a "catch" clause at the very end... but apart from that, no change